plugins/git/README.md

```diff
@@ -238,7 +238,7 @@ These are aliases that have been removed, renamed, or otherwise modified in a wa
 
 | Command                | Description                                                                                              |
 | :--------------------- | :------------------------------------------------------------------------------------------------------- |
-| `grename <old> <new>`  | Rename `old` branch to `new`, including in origin remote                                                 |
+| grename \<old> \<new>  | Rename `old` branch to `new`, including in origin remote                                                 |
```

Comment:
Removed the code formatter, so it is the same style as the rest of the table.

By the was should I go for ordering the functions by name?

Reply:
If you want to do it now and then we merge +1

Reply:
Yes, I will do it.